packages/core/src/Services/socket-general.js

```diff
@@ -307,21 +307,25 @@ export default BinarySocketGeneral;
 const ResponseHandlers = (() => {
     const websiteStatus = response => {
         if (response.website_status) {
-            const is_available = !BinarySocket.isSiteDown(response.website_status.site_status);
-            if (is_available && BinarySocket.getAvailability().is_down) {
-                window.location.reload();
-                return;
-            }
-            const is_updating = BinarySocket.isSiteUpdating(response.website_status.site_status);
-            if (is_updating && !BinarySocket.getAvailability().is_updating) {
-                // the existing connection is alive for one minute while status is updating
-                // switch to the new connection somewhere between 1-30 seconds from now
-                // to avoid everyone switching to the new connection at the same time
-                const rand_timeout = Math.floor(Math.random() * 30) + 1;
+            const { site_status } = response.website_status;
+            const isServerDown = site_status === 'down' || site_status === 'updating';
```

Comment:
Do we need to have cuttoff on BE Updating too? 🤔

Reply:
yep, that was part of the request

Reply:
`website_status` does not get affected by blockRequest - handled: [here](https://github.com/binary-com/deriv-api/commit/fe3d8794f0de802368ace89fc7706c4dd5410512)